README.md

```diff
@@ -64,6 +64,14 @@ const client = new Client({
 })
 ```
 
+or from within a Pod:
+
+```js
+const Client = require('kubernetes-client').Client
```

Comment:
meh, missing a `;`.

(FWIW, we're moving the code base to standard.js asap, but for now, please add the `;`)